beacon/goclient/proposer.go

```diff
@@ -16,12 +16,19 @@ import (
 
 	apiv1bellatrix "github.com/attestantio/go-eth2-client/api/v1/bellatrix"
 	apiv1capella "github.com/attestantio/go-eth2-client/api/v1/capella"
+
+	"github.com/bloxapp/ssv/logging/fields"
+)
+
+const (
+	batchSize = 500
+	// TODO: This is a reasonable default, but we should probably make this configurable.
```

Comment:
ask @Tomblox1 to make tickets for this

Reply:
Done, I'll resolve this once @Tomblox1 confirms